docker/optimism/optimism.sh

```diff
@@ -95,15 +95,53 @@ e2e_test() {
     e2e_test_verify ${DEPLOYMENT_DIR} $l2_rpc $ADDRESS $e2e_test_request_id
 }
 
+
+e2e_test_fallback() {
+    set +eo pipefail
```

Comment:
What is the reason for this line?

Reply:
we need to let relayer to fail and if we dont do this then whole pipeline is failing.

Reply:
But don't we already set the `pipefail` option and `e` in `common.sh`?

Reply:
Ah, I'm dumb.